### PR TITLE
45 add tenant support

### PR DESCRIPTION
**Pull request by arufanov**
45 add tenant support

### CODE REVIEW

**Comment by arufanov** [comment]
@yurichervoniy 
* FYI

File: src/main/java/com/a5lab/tabr/domain/tenants/Tenant.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import jakarta.validation.constraints.NotBlank;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.DynamicUpdate;
+
+@Entity
+@Table(name = "rings")
```
Comment (yurichervoniy) [inline src/main/java/com/a5lab/tabr/domain/tenants/Tenant.java:19]: table name - ring. I think it's a mistake and typo

File: src/main/resources/database/changelog-master.xml [hunk 2]
```diff
@@ -7,6 +7,7 @@
     <property name="now" value="now()" dbms="h2"/>
     <property name="now" value="current_timestamp" dbms="postgresql"/>
 
+    <include file="migrations/2022_10_0000_create_tenants_table.xml" relativeToChangelogFile="true"/>
```
Comment (yurichervoniy) [inline src/main/resources/database/changelog-master.xml:10]: Let's use append only style. put all new migrations to the end

Comment (arufanov) [reply src/main/resources/database/changelog-master.xml:10]: @yurichervoniy 
* we have already discussed it. After release:)

**Review by yurichervoniy — APPROVED** [review]

**Pull request merged**